src/renderer/components/wallet/Settings.tsx

```diff
@@ -64,111 +45,137 @@ const Settings: React.FC<Props> = (props: Props): JSX.Element => {
+                <Styled.OptionCard bordered={false}>
+                  <Styled.Button sizevalue="xnormal" color="primary" typevalue="outline" round="true" disabled>
                     {intl.formatMessage({ id: 'setting.view.phrase' })}
-                  </StyledButton>
-                </StyledOptionCard>
-              </StyledWalletCol>
-              <StyledWalletCol sm={{ span: 24 }} md={{ span: 12 }}>
-                <StyledOptionCard bordered={false}>
-                  <StyledButton
+                  </Styled.Button>
+                </Styled.OptionCard>
+              </Styled.WalletCol>
+              <Styled.WalletCol sm={{ span: 24 }} md={{ span: 12 }}>
+                <Styled.OptionCard bordered={false}>
+                  <Styled.Button
                     sizevalue="xnormal"
                     color="error"
                     typevalue="outline"
                     round="true"
                     onClick={removeWallet}>
                     {intl.formatMessage({ id: 'wallet.action.remove' })}
-                  </StyledButton>
-                </StyledOptionCard>
-              </StyledWalletCol>
+                  </Styled.Button>
+                </Styled.OptionCard>
+              </Styled.WalletCol>
             </Row>
-          </StyledCard>
-          <StyledSubtitle>{intl.formatMessage({ id: 'setting.client' })}</StyledSubtitle>
-          <StyledCard>
+          </Styled.Card>
+          <Styled.Subtitle>{intl.formatMessage({ id: 'setting.client' })}</Styled.Subtitle>
+          <Styled.Card>
             <Row>
               <Col span={24}>
-                <StyledPlaceholder>{intl.formatMessage({ id: 'setting.midgard' })}</StyledPlaceholder>
+                <Styled.Placeholder>{intl.formatMessage({ id: 'setting.midgard' })}</Styled.Placeholder>
 
-                <StyledClientLabel>
+                <Styled.ClientLabel>
                   {pipe(
                     clientUrl,
                     O.getOrElse(() => intl.formatMessage({ id: 'setting.notconnected' }))
                   )}
-                </StyledClientLabel>
+                </Styled.ClientLabel>
 
-                <StyledPlaceholder>{intl.formatMessage({ id: 'setting.version' })}</StyledPlaceholder>
-                <StyledClientLabel>v{apiVersion}</StyledClientLabel>
-                <StyledClientButton color="warning" size="big" onClick={toggleNetwork}>
-                  Change to {network === 'mainnet' ? 'testnet' : 'mainnet'}
-                </StyledClientButton>
+                <Styled.Placeholder>{intl.formatMessage({ id: 'setting.version' })}</Styled.Placeholder>
+                <Styled.ClientLabel>v{apiVersion}</Styled.ClientLabel>
+                <Styled.Placeholder>{intl.formatMessage({ id: 'common.network' })}</Styled.Placeholder>
+                <Dropdown overlay={networkMenu} trigger={['click']}>
+                  <Row align="middle" style={{ width: '130px' }} onClick={(e) => e.preventDefault()}>
```

Comment:
imo its better to avoid fixed sizes in any situation. maybe its better to set `minWidth` ?

Reply:
It seems we can't do this ^, it will expand the menu items in its width (see screenshot)

![Screenshot from 2020-09-01 18-23-49](https://user-images.githubusercontent.com/61792675/91880850-ed559780-ec80-11ea-8de5-e1921159cd26.png)


But we do need this:

![Screenshot from 2020-09-01 18-28-44](https://user-images.githubusercontent.com/61792675/91880889-fba3b380-ec80-11ea-8019-cf02f76c7bf9.png)

Let me know if you have another idea ...

Reply:
ant calculates dropdown width as 100% of a parent (`Row` component in this case). we can set additional style for Row as
```ts
style={{ display: 'inline-flex', minWidth: 130 }}
```

Reply:
Cool, by using `display: 'inline-flex'`  we don't need a width/minWidth anymore, thx!